subsys/bluetooth/controller/ticker/ticker.h

```diff
@@ -74,6 +48,18 @@
 */
 #define TICKER_USER_OP_T_SIZE	44
 
+#define TICKER_CALL_ID_NONE    0
```

Comment:
These macro definitions are not exposed publicly (At least not in this PR), so I wonder if they should be declared in the public ticker header file.

Reply:
This is "the" ticker header file. These CALL_IDs are for integration with HAL (rather say possible ISR, co-operative, and pre-emptive execution contexts of ticker with kernel/bare-bone).

Reply:
Yes, so I wondered _whether_ they should be declared in this header file.